tool/site/client/src/components/prescription-summary/prescriptionLevelDetails.tsx

```diff
@@ -34,6 +34,10 @@ export function createPrescriptionLevelDetails(
   const detailsProps: PrescriptionLevelDetailsProps = {
     prescriptionId,
     courseOfTherapyType: courseOfTherapyTypeCoding.display,
+    prescriptionTypeCode:
+      medicationRequest.extension.find(
+        e => e.url === "https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType"
+      ).valueCoding.code,
```

Comment:
any chance this `find` could return `undefined`?

Reply:
it shouldn't right, I think that extension is required

Reply:
It's required, but should probably handle the possibility of it missing as users can put in custom prescriptions. Have added defensive coding here